.gitignore

```diff
@@ -21,3 +21,4 @@
 ###< symfony/webpack-encore-pack ###
 
 /yarn-error.log
+.idea
```

Comment:
This should belong to your global .gitconfig, as far as I can see. could you remove this?

Reply:
done